### PR TITLE
test: collections length and documents name validation

### PR DESCRIPTION
**Pull request by anversa-pro**
## Describe your changes
Implement new function to check the saved data in mongoDB and the fetched data from firebase.
## Issue ticket number and link
https://trello.com/c/8s3d1EJa
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: scripts/db-migration.js [hunk 1]
```diff
@@ -66,20 +74,39 @@ const mapEventsToMongoDBDocument = (events, mappedheadquarters) => {
...
     console.error('Error saving data to MongoDB:', error)
     throw new Error('Error saving data to MongoDB')
   }
 }
+
+const verifyDataMigration = (mongoCollectionData, firebaseCollectionData) => {
+  let hasMatch = false
+
+  if (mongoCollectionData.length === firebaseCollectionData.length) {
+    for (let i = 0; i < mongoCollectionData.length; i++) {
```
Comment (FelipeLosada1996) [inline scripts/db-migration.js:102]: I don't like the for

**Review by FelipeLosada1996 — APPROVED** [review]

**Review by jose-sal-y-rosas — APPROVED** [review]
👍🏼

File: scripts/db-migration.js (hunk 1 above)
Comment (FelipeLosada1996) [reply scripts/db-migration.js:102]: You can use other alternatives, the for is not the best https://www.freecodecamp.org/espanol/news/map-reduce-y-filter-de-javascript-funciones-de-arreglo-js-explicadas-con-ejemplos-de-codigo/

Comment (anversa-pro) [reply scripts/db-migration.js]: refactor implemented 👍🏼

**Review by FelipeLosada1996 — APPROVED** [review]

**Pull request merged**